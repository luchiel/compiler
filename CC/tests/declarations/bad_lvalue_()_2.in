int a() { return 4; }
void main()
{
&(a());
}