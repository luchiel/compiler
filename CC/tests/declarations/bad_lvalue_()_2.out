Error: ParseError:4:7 Lvalue required as operand
